tests/integration/012_environment/main.tf

```diff
@@ -20,29 +20,15 @@ resource "env0_environment" "example" {
     value = "value"
   }
   approve_plan_automatically = true
+  revision                   = "master"
 }
 
 data "env0_environment" "test" {
   id = env0_environment.example.id
 }
 
-resource "time_sleep" "wait_30_seconds" {
-  depends_on      = [data.env0_environment.test]
-  create_duration = "30s"
+output "revision" {
+  value = data.env0_environment.test.revision
```

Comment:
`name` isn't reliable as a property of the `data` source, because it is mutually-exclusive with `id`.
There are some scenarios where terraform will ignore the `output` entirely, because of that